modules/bibdk_reservation/BibdkReservationOrderObject.php

```diff
@@ -59,6 +62,22 @@ public function getManifestationIds() {
     return $this->manifestationIds;
   }
 
+  public function setSubtypeOrderIds($ids) {
+    $this->subtypeOrderIds = $ids;
+  }
+
+  public function getSubtypeOrderIds() {
+    return $this->subtypeOrderIds;
+  }
+
+  public function setSubtypeOrder($bool) {
```

Comment:
A nice to have: return $this;
Then we can add this very easily to a single call.